Citrix/citrix-adc/tests/test_aaa.json

```
@@ -0,0 +1,50 @@
+{
+  "input": {
+    "message": "01/24/2025:12:06:37 GMT NetscalerCD07_1 0-PPE-0 : default AAA LOGIN_FAILED 73912122 0 : User testUser - Client_ip 1.2.3.4 - Failure_reason \"External authentication server denied access\" - Browser Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
+  },
+  "expected": {
+    "message": "01/24/2025:12:06:37 GMT NetscalerCD07_1 0-PPE-0 : default AAA LOGIN_FAILED 73912122 0 : User testUser - Client_ip 1.2.3.4 - Failure_reason \"External authentication server denied access\" - Browser Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36",
+    "event": {
+      "category": [
+        "network"
+      ],
+      "code": "LOGIN_FAILED",
+      "dataset": "audit_aaa",
+      "reason": "External authentication server denied access",
+      "type": [
+        "connection"
+      ]
```

Can we change `event.category` and `event.type` to `['authentication']` and `['start']` with `event.outcome: "failure"`?
```suggestion
      "category": [
        "authentication"
      ],
      "code": "LOGIN_FAILED",
      "dataset": "audit_aaa",
      "reason": "External authentication server denied access",
      "outcome": "failure",
      "type": [
        "start"
      ]
```